gce/templates/rr-template.yaml

```diff
@@ -0,0 +1,83 @@
+#cloud-config
+---
+write_files:
+    - path: /home/core/install-rr
+      permissions: 0755
+      owner: root
+      content: |
+        #!/bin/bash
+        set -e
+        set -x
+        cd /home/core/
+
+        # Download calicoctl and make it executable:
+        while ! wget https://github.com/projectcalico/calico-containers/releases/download/v0.19.0/calicoctl -O calicoctl; do
+          echo Failed to download calicoctl
+        done
+        chmod +x ./calicoctl
+
+        while ! docker pull __CLUSTER_PREFIX__-master:5000/routereflector;
+        do
+          echo "Failed to pull route reflector image"
+        done
+
+        while ! ./calicoctl bgp node-mesh off;
+        do
+          echo Failed to turn off BGP mesh.
+        done
+
+        calico_as=$(./calicoctl bgp default-node-as)
+        while ! ./calicoctl bgp peer add $private_ipv4 as $calico_as ;
+        do
+          echo Failed to add global RR BGP peer.
+        done
+
+        while ! docker run --privileged \
+                   --net=host -d \
+                   --name=rr \
+                   -e ETCD_AUTHORITY=127.0.0.1:2379 \
+                   -e IP=$private_ipv4 \
+                   __CLUSTER_PREFIX__-master:5000/routereflector;
+        do
+          echo "Failed to start route reflector"
+        done
+        echo "Started RR"
+
+        # Disable bird logging, which is too verbose by default.
```

Comment:
Hmm, if this hack is needed, we should probably have an issue for it.  (Yes, I'm aware that this was my hack.)